libr/cons/cons.c

```diff
@@ -225,7 +225,10 @@ R_API void r_cons_context_break_push(RConsContext *context, RConsBreak cb, void
 	if (r_stack_is_empty (context->break_stack)) {
 #if __UNIX__ || __CYGWIN__
```

Comment:
as said by pancake, it would be much better if we could avoid all these `#if ....` and just have one function where there's such `#if`. That would probably require to define sigset_t and other macros for those systems that don't have them though.